examples/js/effects/AnaglyphEffect.js

```diff
@@ -130,6 +153,12 @@ THREE.AnaglyphEffect = function ( renderer, width, height ) {
 
 	this.render = function ( scene, camera ) {
 
+		if (_invertRedCyanChanged) {
```

Comment:
I think we could achieve the same by swapping `colorMatrixLeft` and `colorMatrixRight` here instead.